src/main/java/duke/Main.java

```diff
@@ -0,0 +1,37 @@
+package duke;
+
+import java.io.IOException;
+
+import duke.controller.MainWindow;
```

Comment:
Good work avoiding wildcard imports. However, should the duke.controller.MainWindow import be grouped together with the javafx imports?